test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -345,7 +415,11 @@ public void GivenOcelotIsRunningUsingConsulToStoreConfigAndJsonSerializedCache()
                     .AddConsul()
                     .AddConfigStoredInConsul();
             })
-            .Configure(app => { app.UseOcelot().Wait(); });
+            .Configure(app =>
+            {
+                // Turning as async/await some tests got broken
+                app.UseOcelot().GetAwaiter().GetResult();
```

Comment:
Why not await ?

Reply:
The test `should_return_response_200_with_simple_url_when_using_jsonserialized_cache` (at `Ocelot.AcceptanceTests.Configuration.ConfigurationInConsulTests` starts to fail. I'm not sure why.

![image](https://github.com/user-attachments/assets/ab1ee15c-c821-46e8-8936-566440b00197)

Reply:
Ah, it should wait synchronously to setup app pipeline... Yes, this is current bottleneck in the design.
Not an issue.